assets/base.css

```diff
@@ -1445,7 +1445,15 @@ details[open] > .share-button__fallback {
   z-index: 3;
   width: 100%;
   min-width: max-content;
-  box-shadow: 0 0 0 0.1rem rgba(var(--color-foreground), 0.55);
+  border: 0.1rem solid transparent;
+  border-radius: var(--inputs-radius);
+  box-shadow: 0 0 0 var(--inputs-border-width) rgba(var(--color-foreground), var(--inputs-border-opacity));
+  filter: drop-shadow(
+    var(--inputs-shadow-horizontal-offset)
+    var(--inputs-shadow-vertical-offset)
+    var(--inputs-shadow-blur-radius)
+    rgba(var(--color-foreground), var(--inputs-shadow-opacity))
+  );
```

Comment:
I wonder if we should be adding some kinda margin-top based on the border width value here so that the overlap isn't really an issue 🤔 
We could potentially take in consideration the vertical offset of the shadow too.